lms/djangoapps/grades/api/views.py

```diff
@@ -153,8 +155,20 @@ def get(self, request, course_id):
         if isinstance(course, Response):
             return course
 
+        if request.user.username == username:
+            graded_user = request.user
```

Comment:
Why di we need the additional variable `graded_user `?

Reply:
@cmltaWt0 just to emphasize it's not necessarily the user that performs the request, but rather a user to get grades for.